_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,287 @@
+---
+id: b8bb68
+name: Text spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that text spacing set in the `style` attribute is below permissible minimum value or not set to `!important`
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] and has one of the following CSS properties set by the `style` attribute:
+
+- [word-spacing][]
+- [letter-spacing][]
+- [line-height][]
+
+## Expectation 1
+
+The test target does not have the [important flag][] set for the CSS [word-spacing][] property, unless the [computed][] [word-spacing][] is greater than `0.16` times the [computed][] [font-size][] of the test target.
```

Comment:
```suggestion
The test target does not have the [important flag][] set for the CSS [word-spacing][] property, unless the [computed][] [word-spacing][] is at least `0.16` times the [computed][] [font-size][] of the test target.
```

Otherwise, `word-spacing: 0.16em` would not be sufficient as it would have to be strictly greater than 0.16, "at least" meaning "greater than or equal to".